src/modules/tp/entities/tp.entity.ts

```diff
@@ -0,0 +1,26 @@
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";
+import {BaseEntity} from "../../../commons";
+
+
+@Entity('trabajo')
+export class TpEntiy extends BaseEntity {
```

Comment:
Renombrar entidad y archivo, veo que se borraron
```suggestion
export class Assignment extends BaseEntity {
```